src/main.rs

```diff
@@ -131,8 +131,8 @@ fn main() {
 				);
 			});
 			let mut update_timer = QTimer::new_0a();
-			// Call it every second
-			update_timer.set_interval(1000);
+			// Call it every 2 seconds
```

Comment:
i guess this depends on the user, we should make this configurable

Reply:
Fair enough, I'll make it a command-line option.